games/main.tex

```diff
@@ -0,0 +1,1123 @@
+peer signed their claim, third parties can also assume that the sum of all claims in the
+allocation table is the maximum total deposit that can be consolidated.
+
+If the soft channel deposit allocation table is valid,
+we can with complete certainty know that the global deposit in A's contract
+ could cover all outstanding conditional notes handed to B even if all peers were to redeem all outstanding notes.
+Practically this means then that, after receiving a valid allocation table for the epoch from A,
+B has no risk of insolvency when dealing with A's conditional notes.
+
+In sum, using soft channel deposits enables the chequebook owner to allocate and reallocate funds
+as channel deposits in a fairly flexible way without blockchain transaction costs,
+yet provide $100\%$ solvency guarantees on all active conditional bonds to their peers.
+
+
+\section{Service guarantees}
+
+\subsection{Service guarantees}
+\label{sec:courtroom}
+
+In the previous section we introduced tools to compensate service providers.
+While rewards are paid out only if delivery is successful,
+such positive incentivisation may be insufficient if
+explicit guarantees of future delivery are required.
+
+In this section we describe a system of service contracts and dispute resolution.
+Service contracts allow providers (or a set of providers)
+to offer quality and/or delivery guarantees to customers.
+Without loss of generality, we can conceptualise a service contract as a contract
+on the blockchain.
+Nodes can commit to a service contract by registering their address in the contract
+with a transaction or issuing a signed note offchain.
+By committing to this specification providers can be held accountable for non-compliance.
+
+Litigation on the other hand, allows customers to initiate an on-chain court procedure
+in case they suspect foul play. Such accountability is
+crucial for the design of scalable decentralised service economies.
+
+We will describe the trial process for dispute
+resolution that first evaluates the evidence agaist the accused provider.
+The verification is the same as the one used in the process of sanctioning outpayments,
+but once a provider is proven guilty by the court on the grounds of
+conditions specified in the service contract, various punitive measures will be
+automatically enforced. In most cases, this comprises losing some deposit that
+provider previously locked up as collateral to back their promise.%
+%
+\footnote{Instead of or in addition to losing their deposits, nodes would also
+suffer loss of reputation. If any demonstrated violation of terms is
+recorded on the public blockchain, users can expect clean histories as a measure
+of reputation.}
+%
+
+By registering on a service contract and putting up a deposit if required, providers
+swear to play by the rules.
+This deposit is locked up and used later as compensatory insurance transfered to the
+customer, redistributed or burnt.
+
```

Comment:
It is reasonable to require the consumers (clients) to deposit some money upon registration ...so the contract can punished them for the case of false accusation.

Reply:
they do not have to register though. they can indeed be required to send an amount along with the accusation.